release.md

```diff
@@ -0,0 +1,167 @@
+# Ignition Release Process
+
+This documentation covers the releasing process done by the members of the
+Ignition/Gazebo team. It offers a general overview and documentation of the
+procedures and tools used and provide an step by step guide to do released.
+
+#### Table of Contents
+
+[General Overview](#general-overview)
+  * [Platforms Supported](#releasing-platforms-supported)
+  * [Infrastructure](#infrastructure)
+  * [Type of Releases](#type-of-releases)
+  * [Metadata for releasing](#metadata-for-releasing)
+  * [Versioning](#versioning)
+
+[Triggering a Release](#)
+
+[FAQ](#faq)
+
+## General overview
+
+Ignition development supports different ways of installing the software in the
+user system. This document covers the process of creating binaries for different
+distributions starting from a local copy of source code together with
+description and explanations of the context of preparing, creating and releasing
+new software versions.
+
+![general releasing context](releasing/images/general_context.png)
+
+### Releasing platforms supported
+
+The Ignition software supports a large variety of platforms where it can be
+built from source. A small subgroup of them is selected to be the officially
+supported for binaries distribution:
+
+ * **Linux:** .deb packages are being created for different Ubuntu and Debian
+   releases. Ubuntu LTS and Debian stable are officially supported.
+   Architectures supported are typically `amd64`, `arm64` and `armhf` but that
+   combination may not be available for all Ubuntu/Debian/Ignition releases.
+
+ * **MacOSX:** binaries for all active releases of MacOSX are usually created by
```

Comment:
I think the correct term nowadays is `macOS`, see #106.

https://en.wikipedia.org/wiki/Macintosh_operating_systems#Mac_OS_X_/_OS_X_/_macOS

There are several instances below.

Reply:
508d5a5